bench/Libraries/Microsoft.Extensions.Http.Resilience.PerformanceTests/HttpClientFactory.cs

```diff
@@ -24,6 +28,10 @@ public enum HedgingClientType
 
 internal static class HttpClientFactory
 {
+    public const string EmptyClient = "Empty";
+
+    public const string StandardClient = "Standard";
+
```

Comment:
Do these have to be public?

[nit] The blanks line can probably go away.

Reply:
Made these internal, but I want to avoid using magic strings because these are tied to pre-configured named clients.